examples/reader-writer.cc

```diff
@@ -0,0 +1,426 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I would prefer to have this file clang-formatted. I would find it much better readable with consistent linebreaks and indentations as the other code.